src/core/router/identity.h

```diff
@@ -292,9 +292,21 @@ struct XORMetric {
   }
 };
 
+/// @brief Hash identity with appended mod date according to spec
+/// @param ident Identity hash used as the basis for the routing key
+/// @throw std::invalid_argument if supplied a zero-initialized ident
+/// @throw std::ios_base::failure on date formatting failure
+/// @throw CryptoPP::Exception on hashing failure
 IdentHash CreateRoutingKey(
     const IdentHash& ident);
 
+/// @brief Format routing key with mod date according to spec
+/// @param ident Identity hash used as the basis for the routing key
+/// @throw std::invalid_argument if supplied a zero-initialized ident
+/// @throw std::ios_base::failure on date formatting failure
```

Comment:
>/// @throw std::ios_base::failure on date formatting failure

We don't need this. We shouldn't keep track of every throw point that isn't directly related to the function's return or interface. Doing so only clutters the code and documentation, and is a pain to maintain should an implementation change.

Reply:
Wasn't sure about the level of detail to include. I agree with you though, I'll remove it.